app/components/windows/NameSource.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 <modal-layout
-  title="Name Source"
+  :title="options.rename ? 'Reame Source' : 'NameSource'"
```

Comment:
- Should be "Rename" instead of "Reame"
- Add a space between "Name" and "Source"

Reply:
thanks